scripts/ci/dockerfile-assets/bash/extract-environment.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Provide environment variables for static frontends by checking referenced
+# variables in the code and inject them into index.html
+#
+
+set -e
+
+work_dir="/usr/share/nginx/html"
+file="$work_dir/index.html"
+
+placeholder="<!-- environment placeholder -->"
+
+function extract_environment() {
+  env_prefix="SI_PUBLIC_"
```

Comment:
Should the prefix maybe come from an environment variable in order to sync with the ts files?

Reply:
I think that would be more confusing as it would need to be defined for every environment in helm and we want this to be the same in all environments.
And also we want the value of this to be visible for developers.